scripts/us_fed/treasury_constant_maturity_rates/treasury_constant_maturity_rates_import_config.txt

```diff
@@ -0,0 +1,10 @@
+import: {
+    import_name: "treasury_constant_maturity_rates"
+    provenance_url: "https://www.federalreserve.gov/releases/h15/"
+    provenance_description: "U.S. Federal Reserve Constant Maturity Rates"
+    mcf_url: "gs://resolved_mcf/external_tables/treasury_constant_maturity_rates_1/nodes.mcf-00000-of-00001",
+    start_date: "1962-01-02"
+    end_date: "2020-05-28"
```

Comment:
since this seems to be a daily updated file, is there a sentinel end_date that will stay current we can use?

Reply:
Can't find one. I am not sure if this field is required.

Reply:
Let's leave end_date and next_release_date out.